client/src/components/ClubDetails.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+export default function ClubDetails() {
+  // Use useParams to get the clubId from URL parameters
+  const { clubId } = useParams();
+
+  // You can use the clubId to fetch the club details or display them as needed
+  // Example:
+  // const clubDetails = fetchClubDetails(clubId);
+
+  return (
+    <div>
+      <h2>Club Details Page</h2>
+      <p>Club ID: {clubId}</p>
```

Comment:
Data object IDs should be for internal use only, they don't need to know the club ID on the UI page

Reply:
I had the data object id for when I was trying to show the detail description of the club when the user would click the button. I'm still working on the function which is why it is still there.